### PR TITLE
Change search post payload name to 'extras' for HCA CLI naming consis…

### PR DESCRIPTION
**Pull request by mikebaumann**
Change search post payload name to 'extras' for HCA CLI naming consistency.

### CODE REVIEW

File: dss/api/search.py [hunk 1]
```diff
@@ -20,7 +20,8 @@ def list():
     return {"query": query, "results": format_results(request, response)}
 
 
-def post(query: dict):
+def post(extras: dict):
+    query = extras
```
Comment (ttung) [inline dss/api/search.py]: are you sure this is where the query is?  mine shows up as extras['query']

**Comment by Mackey22** [comment]
Swagger spec looks good to me. Deferring to @ttung on the way the data gets submitted at the endpoint.

File: dss/api/search.py (hunk 1 above)
Comment (Mackey22) [inline dss/api/search.py]: `extras['query']` seems correct rather than the above.

**Comment by Mackey22** [comment]
@mikebaumann I need this pr merged for my python bindings functionality. Can you fix Tony's comment above and close this out?

**Comment by mikebaumann** [comment]
Closing as this PR has been superseded by the following PRs:
- Rename extras to json_request_body #371 
- Change POST /search API to use json_request_body with es_query property #377

**Pull request closed without merging**